x-pack/plugins/cases/server/client/utils.ts

```diff
@@ -283,46 +284,31 @@ export const constructQueryOptions = ({
   authorizationFilter,
   from,
   to,
-}: {
-  tags?: string | string[];
```

Comment:
These changes aren't necessary, but I initially was trying to get `tags` and other keyword fields searchable but decided not to continue down that route. I figured we could keep the readability improvements though 🤷‍♂️